### ISSUE TITLE
Fix content src element for PullToRefresh sample

### ISSUE THREAD

**Issue by mimendoza** [blackberry/BB10-WebWorks-Samples#39]
@erikj54

< content src > tag points to index.htm, should be index.html

**Comment by ejzn** [comment]
Good catch. Feel free to fix it up :)

**Comment by LizMyers** [comment]
I'm having some difficulty implementing Pull to Refresh in the context of my app. Am trying to use the feature to refresh a Twitter feed embedded in a screen that is not index.html @mimendoza actually that would be a very useful sample - index.htm w/button to load secondary screen > click btn to push Twitter feed screen which has pull To Refresh embedded. Would appreciate either a further sample like this or a little help with my implementation. Thanks in advance.

**Comment by ejzn** [comment]
@LizMyers what are you having trouble with? The sample is pretty simple. This is the key property you want to be concerned with: -webkit-overflow-scrolling: -blackberry-touch; Have you tried mimicking the #scroller div in the sample inside your second page? You could just poach the CSS directly from this sample.

**Comment by LizMyers** [comment]
I know - it's embarrassingly simple. I've tried everything you suggested. Will you be in Amsterdam? Perhaps it would be easier if i show you. Would be nice though if this could be integrated with Twitter sample though. That's pretty similar to what i need and it's a very common use case.

**Comment by ejzn** [comment]
Well it isn't embarrassingly simple :). I'll take your suggestion to integrate it with the Twitter sample. I'm sure we can come up with something that can help the cause as yes, I agree that it is a common use case. I wish I was going to Amsterdam ;). Are you using BBui.js, I'm wondering if there are some conflicts with the div properties.

**Comment by LizMyers** [comment]
Yes - I'm using BBui.js and BBui.css. I could send my app if it would help understand context. Please let me know what email to use.

**Comment by ejzn** [comment]
Sure you can use the one in my profile: erjohnson@rim.com

**Comment by joickle** [comment]
Was there any luck getting this to work with BBui? I'm having a heck of a time.. lol

**Comment by timwindsor** [comment]
@mimendoza @ctetreault Do you think either of you would be able to try this out in bbUI.js and see what the issues are?

**Comment by splatterb0y** [comment]
I am using it with bbUI.js without problems dont forget to adjust the sizing of the containers accordingly AND more important you have to adjust the line:

``` javascript
 if (this.scrollTop < -140) {
```

The negative pixel value has to fit your use case. I fiddled around with it till it worked for me. It's not easy to get this integrated but it works when you dig a little deeper.
